content/en/guide/v10/typescript.md

```diff
@@ -187,7 +187,7 @@ export class Button extends Component {
 }
 ```
 
-You can restrict event handlers by adding a type annotation for `this` to the function signature as the first argument. This argument will be erased after transpilation.
```

Comment:
This content is an explanation about [this parameters](https://www.typescriptlang.org/docs/handbook/functions.html#this-parameters).
Therefore, This is not appropriate in Preact document.